Matplotlib created a temporary config/cache directory at /tmp/matplotlib-b_6uh9bx because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-dtb_r0mg because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-erysd20j because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-fxmqg0mo because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-gu85ff90 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-2kl7n4bz because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-99kxvgaq because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-zh1zkp1d because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-uum_hotq because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-wzl76op9 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-khh2fkh7 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-2xh41ojf because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-pkctz17t because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-klt2pgdo because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-q7vnct43 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-xnx530of because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-3vvf3w5e because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-f47tv6ai because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-1ci6bl9x because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-3p_5iq35 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-mhmqdw5a because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-i21_pg77 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-ku86gdq2 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-jawmxlm2 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-bbjwnwrz because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-jh3x1x8h because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-4olz2t9s because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-8pim6ggw because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-52plo9n6 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-dy669qhx because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-42licmyx because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-1qerp3_k because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-b4fvopbt because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-pmb66yqa because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-j28nn0jt because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-ujih9yn4 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-psod777a because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-_jsziv_8 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-mrd622b1 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-dpszmete because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-b7rscq3f because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-k053dtmg because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-becuw5vj because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-25l7a_o7 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-ed8a8ath because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-dvz0dmm6 because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-fnynyzlf because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-qnew60yh because the default path (/ccs/home/mrowan/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
:::MLLOG {"namespace": "", "time_ms": 1617340883890, "event_type": "POINT_IN_TIME", "key": "submission_benchmark", "value": "deepcam", "metadata": {"file": "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/utils/mlperf_log_utils.py", "lineno": 56}}
:::MLLOG {"namespace": "", "time_ms": 1617340884057, "event_type": "POINT_IN_TIME", "key": "submission_org", "value": "Umbrella Corp.", "metadata": {"file": "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/utils/mlperf_log_utils.py", "lineno": 59}}
:::MLLOG {"namespace": "", "time_ms": 1617340884058, "event_type": "POINT_IN_TIME", "key": "submission_division", "value": "closed", "metadata": {"file": "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/utils/mlperf_log_utils.py", "lineno": 62}}
:::MLLOG {"namespace": "", "time_ms": 1617340884058, "event_type": "POINT_IN_TIME", "key": "submission_status", "value": "onprem", "metadata": {"file": "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/utils/mlperf_log_utils.py", "lineno": 65}}
:::MLLOG {"namespace": "", "time_ms": 1617340884058, "event_type": "POINT_IN_TIME", "key": "submission_platform", "value": "48xSUBMISSION_PLATFORM_PLACEHOLDER", "metadata": {"file": "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/utils/mlperf_log_utils.py", "lineno": 68}}
:::MLLOG {"namespace": "", "time_ms": 1617340884058, "event_type": "INTERVAL_START", "key": "init_start", "value": null, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 109}}
:::MLLOG {"namespace": "", "time_ms": 1617340884058, "event_type": "POINT_IN_TIME", "key": "cache_clear", "value": null, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 110}}
:::MLLOG {"namespace": "", "time_ms": 1617340884059, "event_type": "POINT_IN_TIME", "key": "seed", "value": 333, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 114}}
:::MLLOG {"namespace": "", "time_ms": 1617340884063, "event_type": "POINT_IN_TIME", "key": "global_batch_size", "value": 48, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 187}}
:::MLLOG {"namespace": "", "time_ms": 1617340884063, "event_type": "POINT_IN_TIME", "key": "opt_name", "value": "LAMB", "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 188}}
:::MLLOG {"namespace": "", "time_ms": 1617340884063, "event_type": "POINT_IN_TIME", "key": "opt_base_learning_rate", "value": 0.001, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 189}}
:::MLLOG {"namespace": "", "time_ms": 1617340884063, "event_type": "POINT_IN_TIME", "key": "opt_learning_rate_warmup_steps", "value": 0, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 190}}
:::MLLOG {"namespace": "", "time_ms": 1617340884063, "event_type": "POINT_IN_TIME", "key": "opt_learning_rate_warmup_factor", "value": 1.0, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 191}}
:::MLLOG {"namespace": "", "time_ms": 1617340884064, "event_type": "POINT_IN_TIME", "key": "opt_epsilon", "value": 1e-08, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 192}}
Constructing DeepLabv3+ model...
Number of output channels: 3
Output stride: 16
Number of Input Channels: 16
Selected optimization level O1:  Insert automatic casts around Pytorch functions and Tensor methods.

Defaults for this optimization level are:
enabled                : True
opt_level              : O1
cast_model_type        : None
patch_torch_functions  : True
keep_batchnorm_fp32    : None
master_weights         : None
loss_scale             : dynamic
Processing user overrides (additional kwargs that are not None)...
After processing overrides, optimization options are:
enabled                : True
opt_level              : O1
cast_model_type        : None
patch_torch_functions  : True
keep_batchnorm_fp32    : None
master_weights         : None
loss_scale             : dynamic
Initialized dataset with  5760  samples.
Initialized dataset with  576  samples.
:::MLLOG {"namespace": "", "time_ms": 1617340887862, "event_type": "POINT_IN_TIME", "key": "train_samples", "value": 5760, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 309}}
:::MLLOG {"namespace": "", "time_ms": 1617340887862, "event_type": "POINT_IN_TIME", "key": "eval_samples", "value": 576, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 314}}
:::MLLOG {"namespace": "", "time_ms": 1617340899381, "event_type": "INTERVAL_END", "key": "init_stop", "value": null, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 335}}
:::MLLOG {"namespace": "", "time_ms": 1617340899382, "event_type": "INTERVAL_START", "key": "run_start", "value": null, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 336}}
:::MLLOG {"namespace": "", "time_ms": 1617340899383, "event_type": "INTERVAL_START", "key": "epoch_start", "value": null, "metadata": {"file": "src/deepCam/train_hdf5_ddp.py", "lineno": 342, "epoch_num": 1, "step_num": 0}}
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
Traceback (most recent call last):
  File "src/deepCam/train_hdf5_ddp.py", line 580, in <module>
    main(pargs)
  File "src/deepCam/train_hdf5_ddp.py", line 352, in main
    outputs = net.forward(inputs)
  File "/ccs/home/mrowan/.local/lib/python3.6/site-packages/apex/parallel/distributed.py", line 560, in forward
    result = self.module(*inputs, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/gpfs/alpine/aph114/scratch/mrowan/ml-performance-benchmark/deepcam/summit/src/deepCam/architecture/deeplab_xception.py", line 449, in forward
    x5 = self.global_avg_pool(x)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 2012, in batch_norm
    _verify_batch_size(input.size())
  File "/sw/summit/open-ce/anaconda-base/envs/open-ce-0.1-0/lib/python3.6/site-packages/torch/nn/functional.py", line 1995, in _verify_batch_size
    raise ValueError('Expected more than 1 value per channel when training, got input size {}'.format(size))
ValueError: Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1])
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
CUDA Hook Library: Failed to find symbol mem_find_dreg_entries, python: undefined symbol: __PAMI_Invalidate_region
